specification/advisor/resource-manager/Microsoft.Advisor/stable/2020-01-01/advisor.json

```diff
@@ -1053,6 +1053,10 @@
           "additionalProperties": {
             "type": "string"
           }
+        },
+        "resourceMetadata": {
+          "$ref": "#/definitions/ResourceMetadata",
```

Comment:
Is 'resourceMetadata' readonly? If its writable then this would qualify as a breaking api version change.

See 'New property added to response':
https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw?e=xEcppo

Reply:
We only support GET call for this reosurce type and this is readonly property